recipients_app/lib/view/widgets/dashboard_item.dart

```diff
@@ -0,0 +1,5 @@
+import "package:flutter/widgets.dart";
+
+abstract class DashboardItem extends StatelessWidget {
```

Comment:
why do we need this file?

Reply:
I've just moved it to widgets as it was added (by you?) in models package. As far as I understand it is widget for dashboard items like question for missing phone and so on?

Reply:
Ah no, sorry. I've created abstract class to make it possible to have different cards on dashboard. So now the DashboardCard and SurveyCard both can be shown on dashboard page as they have one abstract parent.